vshard/replicaset.lua

```diff
@@ -402,6 +415,40 @@ local function replica_on_success_request(replica)
     end
 end
 
+--
+-- Validate instance_name if neeed and make a call.
+--
+local function replica_conn_call(replica, func, args, opts)
```

Comment:
Doesn't look too great, but I suppose, we really need this. Don't think, that `name` will be added to the netbox greeting in any foreseeable future. Up to you, it's not strictly required to allow configuring with names.